core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -123,27 +123,23 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
       var imageTagList = dummyElement.getElementsByTagName(
         'oppia-noninteractive-image');
       for (i = 0; i < imageTagList.length; i++) {
-        var filepathObject = JSON.parse(
+        var filename = JSON.parse(
```

Comment:
filepath (since it's reading off filepath-with-value)

Reply:
Same as above.

Reply:
Done!